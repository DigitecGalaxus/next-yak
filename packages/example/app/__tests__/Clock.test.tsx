/// <reference types="jest" />
import React from "react";
import { render } from "@testing-library/react";
import { Clock } from "../Clock";

test("renders Clock component", () => {
  const { asFragment } = render(<Clock />);
  expect(asFragment()).toMatchSnapshot();
});
