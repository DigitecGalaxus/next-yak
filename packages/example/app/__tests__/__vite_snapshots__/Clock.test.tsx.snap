// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders Clock component 1`] = `
<DocumentFragment>
  <div
    class="
  width: 200px;
  height: 200px;
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  margin: 10px auto;
  perspective: 1000px;
"
  >
    <div
      class="
  width: 100%;
  height: 100%;
  border-radius: 50%;
  position: relative;
  transform-style: preserve-3d;
  transition: transform 2s ease-in-out;
  &:hover {
    transform: rotateX(55deg);
  }
"
    >
      <div
        class="
  width: 10px;
  height: 10px;
  border-radius: 50%;
  background: black;
  position: absolute;
  left: 50%;
  top: 50%;
  transform: translate(-50%, -50%);
  translate: 0 0 40px;
"
      />
      <div
        index="0"
      >
        12
      </div>
      <div
        index="1"
      >
        1
      </div>
      <div
        index="2"
      >
        2
      </div>
      <div
        index="3"
      >
        3
      </div>
      <div
        index="4"
      >
        4
      </div>
      <div
        index="5"
      >
        5
      </div>
      <div
        index="6"
      >
        6
      </div>
      <div
        index="7"
      >
        7
      </div>
      <div
        index="8"
      >
        8
      </div>
      <div
        index="9"
      >
        9
      </div>
      <div
        index="10"
      >
        10
      </div>
      <div
        index="11"
      >
        11
      </div>
      <div />
      <div
        class="
  width: 4px;
  height: 40%;
  background: black;
  translate: 0 0 40px;
"
      />
      <div
        class="
  width: 6px;
  height: 30%;
  background: black;
  translate: 0 0 40px;
"
      />
    </div>
  </div>
</DocumentFragment>
`;
